logging/logger.go

```diff
@@ -30,16 +30,16 @@ import (
 // interface must accept and handle smoothly calls to the Debug, Info, Warn and Error methods where
 // the ctx parameter is nil.
 type Logger interface {
-	// DebugEnabled returns true iff the debug level is enabled.
+	// DebugEnabled returns true if the debug level is enabled.
```

Comment:
I suspect these where intentional: "if and only if" :shrug:

Reply:
never crossed my mind, now I know :)